CHANGELOG.md

```diff
@@ -10,6 +10,17 @@ command line for details.
 
 ## 0.8
 
+### [0.8.1] - 2019-02-27
+
+- Provide better error messages
+- Allow auth scope to be array or strings
+- `GitHubOAuthenticator`: More efficient `org_whitelist` check
+- Use pytest-asyncio instead of pytest-tornado since latter is unmaintained
+- CILogon: New additional_username_claims config for linked identities, fallback to the primary username claim
+- `GitLabOAuthenticator`: New `project_id_whitelist` config to whitelist users who has Developer+ access to the project
+- `GoogleOAuthenticator`: Allow email domains (hosted_domain) to be a list & the field is now mandatory
```

Comment:
Add backticks to hosted_domain `hosted_domain`

Reply:
Done.